pkgs/data/icons/sweet-folders/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "sweet-folders";
+  version = "unstable-2023-03-18";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "Sweet-folders";
+    rev = "b2192ff1412472f036fdf9778c6b9dbcb6c044ec";
+    hash = "sha256-QexfqXH5a1IEhKBRjWSMdrEvThvLRzd4M32Xti1DCGE=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons
+    rm -r img LICENSE *.md
+    cp -r . $out/share/icons/
```

Maybe it would be more maintainable to use inclusion rather than exclusion, in case more metafiles are added in the future?

```suggestion
    cp -r Sweet-* $out/share/icons/
```